templates/shared/issuelist.tmpl

```diff
@@ -67,7 +67,7 @@
 						</a>
 					{{end}}
 					{{if .Ref}}
-						<a class="ref" {{if $.RepoLink}}href="{{$.RepoLink}}{{index $.IssueRefURLs .ID}}"{{else}}href="{{AppSubUrl}}/{{.Repo.OwnerName}}/{{.Repo.Name}}{{index $.IssueRefURLs .ID}}"{{end}}>
+						<a class="ref" {{if $.RepoLink}}href="{{index $.IssueRefURLs .ID}}"{{else}}href="{{index $.IssueRefURLs .ID}}"{{end}}>
```

Comment:
Both cases have the same result; simplify as
```suggestion
						<a class="ref" href="{{index $.IssueRefURLs .ID}}">
```

Reply:
Sure, didn't have a close look since this logic appeared unrelated to my PR :joy: